pdfminer/converter.py

```diff
@@ -723,3 +723,152 @@ def render(item: LTItem) -> None:
     def close(self) -> None:
         self.write_footer()
         return
+
+
+class HOCRConverter(PDFConverter):
+    """
+    Where text is being extracted from a variety of types of PDF within a
+    business process, those PDFs where the text is only present in image
+    form will need to be analysed using an OCR tool which will typically
+    output hOCR. This converter extracts the explicit text information from
+    those PDFs that do have it and uses it to genxerate a basic hOCR
+    representation that is designed to be used in conjunction with the image
+    of the PDF in the same way as genuine OCR output would be, but without the
+    inevitable OCR errors.
+
+    The converter does not handle images, diagrams or text colors.
+
+    In the examples processed by the contributor it was necessary to set
+    LAParams.all_texts to True.
+    """
```

Comment:
Use the docstring to describe the purpose (tagline) and (optionally) the behavior of the class.